pkg/utils/utils.go

```diff
@@ -693,3 +701,45 @@ func isSwitchdev(name string) bool {
 
 	return true
 }
+
+func hasMellanoxInterfacesInSpec(newState *sriovnetworkv1.SriovNetworkNodeState) bool {
+	for _, ifaceStatus := range newState.Status.Interfaces {
+		if ifaceStatus.Vendor == VendorMellanox {
+			for _, iface := range newState.Spec.Interfaces {
+				if iface.PciAddress == ifaceStatus.PciAddress {
+					glog.V(2).Infof("hasMellanoxInterfacesInSpec(): Mellanox device %s (pci: %s) specified in SriovNetworkNodeState spec", ifaceStatus.Name, ifaceStatus.PciAddress)
+					return true
+				}
+			}
+		}
+	}
+	return false
+}
+
+// IsKernelLockdownMode returns true when kernel lockdown mode is enabled
+func IsKernelLockdownMode() bool {
+	// Check 2 path to allow using with chroot
+	for _, path := range []string{"/host/sys/kernel/security/lockdown", "/sys/kernel/security/lockdown"} {
+		out, err := RunCommand("cat", path)
+		glog.V(2).Infof("IsKernelLockdownMode(): %s, %+v", out, err)
+		if err == nil && strings.Contains(out, "[integrity]") {
```

Comment:
actually both `[integrity]` and `[confidentiality]` are considered lockdown
so better search for anything other than `[none]`